### DISCUSSION TITLE
GET CUSTOM FIELD INFO INTO CONFIG TEMPLATE

### DISCUSSION THREAD

**Discussion by lucasfontini** [netbox-community/netbox#16634] [Q&A]
I have a custom field for devices and this field is an object that linked with dcim>devices. I`m struggling to get this data into my config_template 

field name connectedto

config template 
{{ device.custom_fields["connectedto"] }}
{{ device.custom_fields.connectedto }}

   "tags": [],
    "custom_fields": {
        "ConnectedTo": {
            "id": 2056,
            "url": "https://netbox-sandbox.master.test.com/api/dcim/devices/2056/",
            "display": "MYDEVICE",
            "name": "MYDEVICE"
        },
        "another_field": 17

**Comment by candlerb** [comment]
`connectedto` is not the same as `ConnectedTo`. Also it's `custom_field_data` (or `cf`), not `custom_fields`.

```
{{ device.custom_field_data["ConnectedTo"] }}
{{ device.cf.ConnectedTo }}
```

**Comment by jonasnieberle** [comment]
You can access the device custom fields with device.cf.custom_field_name_1

![image](https://github.com/netbox-community/netbox/assets/100405740/d318abac-8444-4012-a797-3a335808e948)

See: https://netboxlabs.com/docs/netbox/en/stable/customization/custom-fields/
